cdp/cdp_parser.py

```diff
@@ -107,17 +107,17 @@ def get_orig_parameters(self, check_values=False, argparse_vals_only=True):
         """
         self._parse_arguments()
         
-        if not self.__args_namespace.parameters:
```

Comment:
This line checks that the parameters defined in the file, e.g. `params.py` below, are processed and in this namespace.

A namespace was used because when `parse_args()` is called more than one time, from the second time onwards, it deletes all of the args it read in. This was the case when a user would call `view_args()`, which would just return the value of `parse_args()`. Thus, I saved the values of `parse_args()` into a variable.